src/network.js

```diff
@@ -38,12 +40,12 @@ class Network {
     const cb = (err) => setImmediate(() => callback(err))
 
     if (this.isOnline) {
-      return cb(new Error('Network is already running'))
+      return cb(errcode(new Error('Network is already running'), 'ERR_NETWORK_ALREADY_RUNNING'))
     }
 
     // TODO add a way to check if switch has started or not
     if (!this.dht.isStarted) {
-      return cb(new Error('Can not start network'))
+      return cb(errcode(new Error('Can not start network'), 'ERR_CANNOT_START_NETWORK'))
```

Comment:
Keep this in line with the other network errors.
```suggestion
      return cb(errcode(new Error('Can not start network'), 'ERR_NETWORK_CANNOT_START'))
```